### DISCUSSION TITLE
nvim-tree hijacks <C-i>

### DISCUSSION THREAD

**Discussion by elliotwesoff** [nvim-tree/nvim-tree.lua#1772] [Q&A]
Hi there. I use `<C-o>` and `<C-i>` quite often to jump back and forward between locations, but since installing nvim-tree, `<C-i>` only opens and brings focus to the tree. I don't have a keymapping set (that I know of) to do this, so is there a way to restore `<C-i>`'s original behavior? Thanks!

**Comment by alex-courtis** [comment]
`<Tab>` (`<C-i>`) is mapped by default to the `<preview>` action.

You can remove this mapping:
```lua
    remove_keymaps = {
      '<Tab>',
    },
```

Alternatively, you could map another action to `<Tab>`.

See `:help nvim-tree-mappings`
